src/spritekit.cs

```diff
@@ -730,13 +736,17 @@ partial interface SKSpriteNode : SKWarpable {
 		[Export ("scaleToSize:")]
 		void ScaleTo (CGSize size);
 
+		[iOS (10,0), Mac (10, 12)]
+		[Export ("attributeValues", ArgumentSemantic.Copy)]
+		NSDictionary<NSString, SKAttributeValue> AttributeValues { get; set; }
+
 #if XAMCORE_4_0
-		[iOS (9,0), Mac(10,11)]
+		[iOS (10,0), Mac(10,12)]
```

Comment:
We check those are runtime so, in general ours are better than Apple.
When/if updating them we need to dig up the _truth_ about the API availability. That means, minimally confirming the new values as correct.